src/storage-client/src/controller.rs

```diff
@@ -278,7 +278,12 @@ pub trait StorageController: Debug + Send {
     /// In the future, this API will be adjusted to support active replication
     /// of storage instances (i.e., multiple replicas attached to a given
     /// storage instance).
-    fn connect_replica(&mut self, id: StorageInstanceId, location: ClusterReplicaLocation);
+    fn connect_replica(
+        &mut self,
+        id: StorageInstanceId,
+        location: ClusterReplicaLocation,
+        cluster_size: Option<String>,
```

Comment:
docs on what this is (the human-readable `string` size if there is one)

Reply:
done